build/gradle/commitAndProductionStage.gradle

```diff
@@ -826,6 +844,34 @@ def packageServer(productOutputPath) {
     }
 }
 
+/**
+ * Method executed only for OS X. 
+ * To run the HALE studio in mac, JRE must be bundled within the HALE studio app
+ * and its location must be added in the HALE.ini file.
+ * This method is a work around to modify HALE.ini file.
+ * This method reads HALE.ini file stored under "~/Eclipse.app/Contents/Eclipse/HALE.ini",
+ * then appends "-vm" arguments along with the JRE location bundled within the application.
+ * Finally, the changes are written back to the "HALE.ini" file.
+ * filepath HALE.ini file path
+ */
+def updateHaleIniFileWithJrePath(filepath) {
+
+	File file = new File(filepath)
+	def fileContent = file.getText();
+
+	def indexOfVmargs = fileContent.indexOf("-vmargs")
```

Comment:
I think it would be a bit cleaner to read the lines, but as I also would not expect `-vmargs` to show up in a different context in the ini file I think it is OK like this.

Reply:
No modification done in the logic of reading the file. However, made a little bit change when inserting strings to make the code a bit cleaner.